File: src/gametest/java/org/trp/shincolle/gametest/MinimalGameTestBootstrapTest.java
package org.trp.shincolle.gametest;

import net.minecraft.gametest.framework.GameTest;
import net.minecraft.gametest.framework.GameTestHelper;
import net.neoforged.neoforge.gametest.GameTestHolder;
import net.neoforged.neoforge.gametest.PrefixGameTestTemplate;

@GameTestHolder("shincolle")
@PrefixGameTestTemplate(false)
public final class MinimalGameTestBootstrapTest {
    private MinimalGameTestBootstrapTest() {
    }

    @GameTest(template = "empty")
    public static void emptyTemplateShouldLoad(GameTestHelper helper) {
        helper.succeed();
    }
}

File: src/gametest/kotlin/org/trp/shincolle/gametest/MinimalGameTestBootstrapTest.kt
package org.trp.shincolle.gametest

import net.minecraft.gametest.framework.GameTest
import net.minecraft.gametest.framework.GameTestHelper
import net.neoforged.neoforge.gametest.GameTestHolder
import net.neoforged.neoforge.gametest.PrefixGameTestTemplate

@GameTestHolder("shincolle")
@PrefixGameTestTemplate(false)
class MinimalGameTestBootstrapTest {
    private constructor()

    @GameTest(template = "empty")
    fun emptyTemplateShouldLoad(helper: GameTestHelper) {
        helper.succeed()
    }
}
